src/test_typing_extensions.py

```diff
@@ -4763,34 +4820,34 @@ class Base(TypedDict, closed=True):
         self.assertEqual(Base.__extra_items__, ReadOnly[Union[str, None]])
         self.assertTrue(Base.__closed__)
 
-        class Child(Base):
+        class Child(Base, closed=True):
             a: int
             __extra_items__: int
 
-        self.assertEqual(Child.__required_keys__, frozenset({'a', "__extra_items__"}))
+        self.assertEqual(Child.__required_keys__, frozenset({'a'}))
         self.assertEqual(Child.__optional_keys__, frozenset({}))
         self.assertEqual(Child.__readonly_keys__, frozenset({}))
-        self.assertEqual(Child.__mutable_keys__, frozenset({'a', "__extra_items__"}))
-        self.assertEqual(Child.__annotations__, {"__extra_items__": int, "a": int})
-        self.assertEqual(Child.__extra_items__, ReadOnly[Union[str, None]])
-        self.assertFalse(Child.__closed__)
+        self.assertEqual(Child.__mutable_keys__, frozenset({'a'}))
+        self.assertEqual(Child.__annotations__, {"a": int})
+        self.assertEqual(Child.__extra_items__, int)
+        self.assertTrue(Child.__closed__)
```

Comment:
```suggestion
        self.assertEqual(Child.__closed__, None)
```

We should test the case of a child with a closed parent (2/2)